Makefile

```diff
@@ -42,5 +44,5 @@ watch-serve:
 
 .PHONY: serve
 serve:
-	go install github.com/fogleman/serve
+	go install github.com/fogleman/serve@latest
```

Comment:
I believe that nothing should be (surprise) installed as part of a `make serve` (or similar).

That should be in the prerequisites.

Admittedly, `ci` needs it, but that can be done in the `ci/scripts` explicitly

Reply:
mmm I did think something along these lines, it's not needed for `ci` though I think as we just build the flat files? 

What would your recommended approach be? Have it in the pre-reqs in the readme and then have it fail if they don't have it?

Reply:
in that case, yes (pre-reqs and fail)

Reply:
could move the install line to (e.g) `make install-pre-reqs` or similar